tests/unit/components/UI/Alert/DialogBox.spec.js

```diff
@@ -11,19 +15,66 @@ describe("DialogBox.vue", () => {
     const title = "Dialog Title";
     const wrapper = mount(DialogBox, {
       props: {
-        title,
+        title: title,
       },
     });
-    expect(wrapper.find("p").text()).toBe(title);
+
+    expect(wrapper.get('[data-test="title"]').text()).toBe(title);
   });
 
   it("renders description correctly", () => {
     const description = "Dialog Title";
     const wrapper = mount(DialogBox, {
       props: {
-        description,
+        description: description,
+      },
+    });
+
+    expect(wrapper.get('[data-test="description"]').text()).toBe(description);
+  });
+
+  it("renders image correctly", () => {
+    const wrapper = mount(DialogBox, {
+      props: {
+        iconConfig: {
+          enabled: true,
+          name: "check",
+          class: "w-12 h-12",
+        },
+      },
+      global: {
+        stubs: {
+          InlineSvg: InlineSvg,
+        },
       },
     });
-    expect(wrapper.find("p").text()).toBe(description);
+    expect(wrapper.get('[data-test="icon"]').html()).toContain("img");
+    expect(wrapper.get('[data-test="icon"]').classes()).toEqual(
+      expect.arrayContaining(["w-12", "h-12"])
+    );
+  });
+
+  it("confirm button present", () => {
+    const wrapper = mount(DialogBox);
+    expect(wrapper.get('[data-test="confirmButton"]').text()).toBe("Yes");
+  });
+
+  it("cancel button present", () => {
+    const wrapper = mount(DialogBox);
+    expect(wrapper.get('[data-test="cancelButton"]').text()).toBe("No");
+  });
+
+  it("confirm click works", () => {
+    const wrapper = mount(DialogBox);
+    wrapper.get('[data-test="confirmButton"]').trigger("click");
```

Comment:
Shouldn't we await for this trigger? Everywhere on the docs it's written that the best practice is to await for a trigger to complete